kyuubi-common/src/main/scala/org/apache/kyuubi/service/TFrontendService.scala

```diff
@@ -114,6 +114,12 @@ abstract class TFrontendService(name: String)
     checkInitialized()
     val host = (conf.get(FRONTEND_ADVERTISED_HOST), serverHost) match {
       case (Some(advertisedHost), _) => advertisedHost
+      case (None, Some(h)) if (h.equals("0.0.0.0")) =>
+        if (conf.get(FRONTEND_CONNECTION_URL_USE_HOSTNAME)) {
+          Utils.findLocalInetAddress.getCanonicalHostName
```

Comment:
this function is moved to `JavaUtils` in https://github.com/apache/kyuubi/commit/315adda3539bcfab2bb2b9cb2ac6882f8444721c, could you please rebase your branch to latest master?